washingtonpost/klaxon
# NAPPS-1348: add WP logo

Adds WP logo to Klaxon. If a file named `logo.svg` is present in `app/assets/images/`, then the top left of the app should appear like:

<img width="719" alt="Screenshot 2023-06-20 at 8 28 32 PM" src="https://github.com/WPMedia/klaxon/assets/26312638/b9030380-bde5-47a0-878a-2f08d15191ec">

<img width="831" alt="Screenshot 2023-06-20 at 8 27 35 PM" src="https://github.com/WPMedia/klaxon/assets/26312638/24399c13-0e0b-4c2a-a064-376634980e72">

Otherwise, the top left should look like:

<img width="721" alt="Screenshot 2023-06-20 at 8 26 30 PM" src="https://github.com/WPMedia/klaxon/assets/26312638/bf546716-d6f3-4f55-91ab-43b88ec93aa7">

## Ticket

[NAPPS-1348]

## Testing steps
- If you've already run Klaxon locally before, spin up your container and observe that there is just the Klaxon logo in the top left of the app.
- Rebuild your container (`docker-compose down && docker-compose build && docker-compose up`), and observe that the top left now reads `Klaxon for WP`
- If you hadn't run Klaxon locally before, you can now rename `app/assets/images/logo.svg` to be anything you like. Then, rebuild the container and observe that the app works fine without the logo.
- Review docs for clarity


[NAPPS-1348]: https://arcpublishing.atlassian.net/browse/NAPPS-1348?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
